script.js

```diff
@@ -19,7 +98,8 @@ class Store {
     const books = Store.getBooks();
     books.push(book);
     localStorage.setItem('books', JSON.stringify(books));
-    document.location.reload();
+    // window.alert('record added');
+    // used for demonstration purpose
   }
```

Comment:
- [ ] Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.

Reply:
Thank you for your guidance.